products/compute/terraform.yaml

```diff
@@ -321,6 +321,38 @@ overrides: !ruby/object:Provider::ResourceOverrides
         exclude: false
   TargetPool: !ruby/object:Provider::Terraform::ResourceOverride
     exclude: true
+    examples: |
+      ```hcl
+      resource "google_compute_target_pool" "default" {
+        name = "instance-pool"
+
+        instances = [
+          "us-central1-a/myinstance1",
+          "us-central1-b/myinstance2",
+        ]
+
+        health_checks = [
+          "${google_compute_http_health_check.default.name}",
+        ]
+      }
+
+      resource "google_compute_http_health_check" "default" {
+        name               = "default"
+        request_path       = "/"
+        check_interval_sec = 1
+        timeout_sec        = 1
+      }
+      ```
+    properties:
+      id: !ruby/object:Provider::Terraform::PropertyOverride
+        exclude: true
+      region: !ruby/object:Provider::Terraform::PropertyOverride
+        required: false # the provider-default value will be used if not specified
```

Comment:
Does this work even though region is a parameter (not a property)?

Reply:
Yes, `PropertyOverride` can override parameters or properties. 

https://github.com/GoogleCloudPlatform/magic-modules/blob/master/provider/resource_overrides.rb#L140
https://github.com/GoogleCloudPlatform/magic-modules/blob/master/api/resource.rb#L263